tests/handlers/test_base_handler.py

```diff
@@ -8,45 +8,44 @@
 # http://www.opensource.org/licenses/mit-license
 # Copyright (c) 2011 globo.com thumbor@googlegroups.com
 
-import tempfile
-import shutil
-from os.path import abspath, join, dirname
 import os
-from datetime import datetime, timedelta
-import pytz
+import shutil
 import subprocess
+import tempfile
+from datetime import datetime, timedelta
 from json import loads
+from os.path import abspath, dirname, join
+from urllib.parse import quote
 
+import pytest
+import pytz
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [third party import "import pytz" should be placed before "import pytest"](https://app.codacy.com/manual/heynemann/thumbor/pullRequest?prid=4967909)